params/protocol_params.go

```diff
@@ -146,15 +146,28 @@ func GetMaximumExtraDataSize() uint64 {
 	return BFTMaximumExtraDataSize
 }
 
+// As istanbulHF items are brought to klaytn code, 0x09-0x0b contracts are moved to 0x3fd-0x3ff.
+// For this reason, contracts deployed before HF use old precompiledContract map so it can work even after HF.
+// To Record the deployment time, the MSB of CodeFormat is changed to a istanbulHF field.
+// case 1. CodeFormat [0 0 0 0 0 0 0 0]. MSB is 0, so the contract is deployed before istanbulHF.
+// case 2. CodeFormat [1 0 0 0 0 0 0 0]. MSB is 1, so the contract is deployed after istanbulHF.
+
 type CodeFormat uint8
 
+const (
+	IstanbulHFFieldZeroCF = 127 // 0b01111111: hardfork field value is zero, but codeFormat field values are one
+	CodeFormatFieldZeroCF = 128 // 0b10000000: hardfork field value is one, but codeFormat field values are zero
+)
+
 const (
 	CodeFormatEVM CodeFormat = iota
 	CodeFormatLast
```

Comment:
We need a comment ensuring that "CodeFormatLast should be smaller than CodeFormatFieldZeroCF".

Reply:
As we discussed on offline, size of the codeFormat field is fixed, 4 bit, so CodeFormatLast should be smaller than 16. Anyway, the comment which contains size of each field is added to the above of `type codeInfo uint8`( Naming is really difficult.. For now, I named it `codeInfo`).